### PR TITLE
PY2: UUID accepts both unicode and str

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: stdlib/2and3/uuid.pyi [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
 import sys
-from typing import Tuple, Optional, Any
+from typing import Tuple, Optional, Any, Union
 
 # Because UUID has properties called int and bytes we need to rename these temporarily.
 _Int = int
 _Bytes = bytes
 _FieldsType = Tuple[int, int, int, int, int, int]
+if sys.version_info >= (3,):
```
Comment (JelleZijlstra) [inline stdlib/2and3/uuid.pyi]: Just use `typing.Text`.

Comment (mr-c) [reply stdlib/2and3/uuid.pyi]: On PY2, `typing.Text` is just `unicode`, the point of this PR is that both unicode and str are acceptable.

My approach here is copied from `stdlib/2and3/argparse.pyi`, it is also found in `stdlib/2and3/datetime.pyi`, `stdlib/2and3/optparse.pyi`, `stdlib/2and3/rlcompleter.pyi`, and `third_party/2and3/requests/api.pyi`.

If there is a better way then I'll use that here and send a companion PR to update the other type stubs.

Comment (srittau) [reply stdlib/2and3/uuid.pyi]: In Python 2, `unicode` is a super type of `str`, i.e. `str` is accepted where `unicode` is accepted. This makes the following annotations equivalent for arguments (in Py2): `Text`, `unicode`, `Union[str, unicode]`.

Comment (mr-c) [reply stdlib/2and3/uuid.pyi]: @srittau Okay, I just saw https://github.com/python/peps/pull/302/files

Shall I make a separate PR for those other users of `union[str, unicode]`?

**Pull request merged**